src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
> why is it even allowed to `import Test` a whole module?

It only brings in the module name. `importall Test` imports all of its functions (I forget whether just the exported ones, or absolutely everything) for extension, and is sketchy enough that there's rarely a good reason to use it.